### DISCUSSION TITLE
Announcing Tailwind CSS v1.9!

### DISCUSSION THREAD

**Discussion by adamwathan** [tailwindlabs/tailwindcss#2552] [Show and tell]
We just released Tailwind CSS v1.9 which adds support for configuration presets, useful new CSS grid utilities, extended border radius, rotate, and skew scales, helpful accessibility improvements, and more!

[**Read about the release on the Tailwind CSS blog &rarr;**](https://blog.tailwindcss.com/tailwindcss-1-9)

Here's an overview of what's included:

#### New features

- Add new `presets` config option ([#2474](https://github.com/tailwindlabs/tailwindcss/pull/2474))
- Scaffold new `tailwind.config.js` files with available `future` flags commented out ([#2379](https://github.com/tailwindlabs/tailwindcss/pull/2379))
- Add `col-span-full` and `row-span-full` ([#2471](https://github.com/tailwindlabs/tailwindcss/pull/2471))
- Make `outline` configurable, `outline-none` more accessible by default, and add `outline-black` and `outline-white` ([#2460](https://github.com/tailwindlabs/tailwindcss/pull/2460))
- Add additional small `rotate` and `skew` values ([#2528](https://github.com/tailwindlabs/tailwindcss/pull/2528))
- Add `xl`, `2xl`, and `3xl` border radius values ([#2529](https://github.com/tailwindlabs/tailwindcss/pull/2529))
- Add new utilities for `grid-auto-columns` and `grid-auto-rows` ([#2531](https://github.com/tailwindlabs/tailwindcss/pull/2531))
- Promote `defaultLineHeights` and `standardFontWeights` from experimental to future

#### Fixed

- Don't escape keyframe values ([#2432](https://github.com/tailwindlabs/tailwindcss/pull/2432))
- Use `word-wrap` instead of `overflow-wrap` in `ie11` target mode ([#2391](https://github.com/tailwindlabs/tailwindcss/pull/2391))

#### Experimental

- Add experimental `2xl` breakpoint ([#2468](https://github.com/tailwindlabs/tailwindcss/pull/2468))
- Rename `{u}-max-content` and `{u}-min-content` utilities to `{u}-max` and `{u}-min` in experimental extended spacing scale ([#2532](https://github.com/tailwindlabs/tailwindcss/pull/2532))
- Support disabling dark mode variants globally ([#2530](https://github.com/tailwindlabs/tailwindcss/pull/2530))

Check it out and let us know what you think!

**Comment by brandonpittman** [comment]
I've got `experimental: 'all"` in my config and I'm still getting w-max-content in my output.

**Reply by adamwathan** [reply]
Do you have the Tailwind UI plugin installed?

**Reply by brandonpittman** [reply]
@adamwathan Yes…

The UI plugin seems to clobber a lot from the base library. The core changelogs need a reminder/disclaimer. I keep forgetting this.

**Reply by adamwathan** [reply]
Tailwind UI is essentially an alternate base config file yeah. The work we’re doing right now is to make that plugin unnecessary by adding everything directly to Tailwind though, so in a few weeks when 2.0 is released everything will be a lot simpler.

**Reply by brandonpittman** [reply]
@adamwathan I’m thinking I might just extract the different bits out of the UI plugin tomorrow and stick it in another plugin so I can remove it.
